src/components/form/WizardRepositoryTable.js

```diff
@@ -71,17 +71,25 @@ const WizardRepositoryTable = (props) => {
     }));
   };
   const locationProp = props?.label?.props?.locationProp;
+  const historyProp = props?.label?.props?.historyProp;
+  const navigateProp = props?.label?.props?.navigateProp;
+  const contentRepoLinkProp = props?.label?.props?.contentRepoLinkProp;
+  const contentRepoLink =
+    contentRepoLinkProp === undefined ? true : contentRepoLinkProp;
   return (
     <>
       {!isLoading && !data?.count > 0 ? (
         <EmptyState
+          target="_blank"
           icon="repository"
           title="No custom repositories available"
           body="Add custom repositories to build RHEL for Edge images with additional packages."
           primaryAction={{
             text: 'Custom repositories',
-            href: paths.repositories,
+            href: contentRepoLink ? paths.repositories : '/insights/content',
```

Comment:
This is valid  the when "edge-management.custom_repos_ui" feature is enabled, what value href must be when "edge-management.custom_repos_ui" is disabled ?
 I think it should be paths.repositories instead of "'/insights/content'" when it's disabled

Reply:
@ldjebran when its disabel it should be like that -'/preview/edge/repositories'
now its woking perfect